src/Docfx.Plugins/OutputFileCollection.cs

```diff
@@ -35,5 +35,5 @@ public class OutputFileCollection : ObservableDictionary<string, OutputFileInfo>
 
     #endregion
 
-    public bool ContainsValue(OutputFileInfo value) => Values.Contains(value);
+    public bool ContainsValue(OutputFileInfo value) => ContainsValue(value);
```

Comment:
Just lurking and noticed this looking suspiciously recursive (probably a false positive from `CA1841`)

Reply:
Thanks for pointing that out.
It's my fault for manually editing the code.

I've reverted code and suppress CA1841 message with commit (https://github.com/dotnet/docfx/pull/9224/commits/fef21e68306cc6bf659c9e40403461fd1e1bcc35)